pokemongo_bot/__init__.py

```diff
@@ -602,8 +602,10 @@ def _setup_api(self):
         # chain subrequests (methods) into one RPC call
 
         self._print_character_info()
+        # self.api.activate_signature("/home/pi/PokemonGo-Bot/encrypt.so")
 
-        self.api.activate_signature("encrypt.so")
+        cwd = os.getcwd()
```

Comment:
I don't think you want cwd, I think you want `os.path.join` with `os.path.dirname(__file__)`